src/gbinder_rpc_protocol.c

```diff
@@ -33,6 +33,7 @@
 #include "gbinder_rpc_protocol.h"
 #include "gbinder_reader.h"
 #include "gbinder_writer.h"
```

Comment:
`gbinder_writer_p.h` includes `gbinder_writer.h` (which is generally the case for private headers with public counterparts), it doesn't make sense to include both

Reply:
Removed the include after changing code to use writer functions instead of writer_data functions.